paparazzi/paparazzi/build.gradle

```diff
@@ -28,46 +28,46 @@ dependencies {
   // TODO: Use Gradle transforms to extract/rename classes.jar's from AARs
   api files('libs/compose-runtime-1.2.1.jar')
   compileOnly files('libs/compose-ui-1.2.1.jar')
-  compileOnly deps.androidx.lifecycleCommon
+  compileOnly libs.androidx.lifecycle.common
   compileOnly files('libs/lifecycle-runtime-2.5.0.jar')
   compileOnly files('libs/savedstate-1.2.0.jar')
```

Comment:
left these here because they depend on files in the repo, so they can't be just "bumped", but also it's not possible to refer to "paths" in toml.